docs/instructions.md

```diff
@@ -34,6 +34,65 @@ TRAIN_FUNCTION=<function to invoke to start model training>
 10. to run the predict container:
 > `docker run --rm  -v $(PROJECT_ROOT_DIR)/models:/action/models -e MODElNAME=${MODEL_NAME} -p 5111:5000 -it ${PREDICT_CONTAINER_NAME}`
 
+## Model Predict: Request/Response
+
+The predict endpoint takes an array of instances containing an array of feature values.
+```
+curl -X POST \
+ http://localhost:5111/<route> \
+ -H 'Content-Type: application/json' \
+ -d '{
+"payload": { "instances": [[7,107,74,0,0,29.6,0.254,31]] }
+}'
+```
+`route` is defined using .s2i/environment file
+
```

Comment:
Add clarification that the predictions have to be in the same order as the instances